src/roiextractors/extractors/suite2p/suite2psegmentationextractor.py

```diff
@@ -65,13 +65,13 @@ def get_available_planes(cls, folder_path: PathType):
         plane_names: list
             List of plane names.
         """
+        from natsort import natsorted
+
         folder_path = Path(folder_path)
         prefix = "plane"
-        plane_names = []
-        for item in os.listdir(folder_path):
-            if item.startswith(prefix):
-                plane_names.append(item)
-
+        plane_paths = natsorted(folder_path.glob(pattern=prefix + "*"))
+        assert len(plane_paths), f"No planes found in '{folder_path}'."
+        plane_names = [plane_path.stem for plane_path in plane_paths]
```

Comment:
@alessandratrapani  Thanks for helping with this PR! I refactored this part to make sure we are returning the name of the planes in sorted order.